src/internal/connector/sharepoint/data_collections.go

```diff
@@ -180,9 +192,10 @@ func collectPages(
 	spcs := make([]data.BackupCollection, 0)
 
 	// make the betaClient
+	// Need to receive From DataCollection Call
 	adpt, err := graph.CreateAdapter(creds.AzureTenantID, creds.AzureClientID, creds.AzureClientSecret)
 	if err != nil {
-		return nil, errors.Wrap(err, "adapter for betaservice not created")
+		return nil, fmt.Errorf("unable to create adapter w/ env credentials")
```

Comment:
nit: since we're not passing any format args this can be just `errors.New` from either `pkg/errors` or stdlib `errors`. Probably a preference for `pkg/errors` though as it gives stack trace information
```suggestion
		return nil, errors.New("unable to create adapter w/ env credentials")
```